test/controllers/publishers/omniauth_callbacks_controller_test.rb

```diff
@@ -785,6 +785,7 @@ def auth_hash(options = {})
       request_login_email(publisher: publisher)
       url = publisher_url(publisher, token: publisher.reload.authentication_token)
       get(url)
+      puts response.body
```

Comment:
nitpick- I don't think you meant to leave this in here.

Reply:
thanks!